src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { IsError } from '../../types/IsError';
+
+interface Props {
+  isError: IsError,
+  setIsError: (IsError: IsError) => void,
+  isErrorDefault: IsError,
+}
+
+export const ErrorNotification: React.FC<Props> = ({
+  isError,
+  setIsError,
+  isErrorDefault,
+}) => {
+  const error = Object.entries(isError);
+  const errorName = error.find(el => el[1] === true);
+  const justErrorName = errorName && errorName[0].slice(0, -5);
+
+  const handleClose = () => {
+    setIsError(isErrorDefault);
+  };
+
+  useEffect(() => {
+    setTimeout(() => setIsError(isErrorDefault), 2000);
+  }, []);
```

Comment:
You should clear the timeout on the component unmount https://bobbyhadz.com/blog/react-cleartimeout